cmd/config/virt-udn-density/vm-client.yml

```diff
@@ -0,0 +1,58 @@
+apiVersion: kubevirt.io/v1
+kind: VirtualMachine
+metadata:
+  name: virt-client-{{.Iteration}}
+  labels:
+    kubevirt.io/os: cirros
```

Comment:
Can you update both "kubevirt.io/os" labels to something like 'fedora'? it could be helpful for metadata collection